pkg/ebpf/probes/probes.go

```diff
@@ -52,10 +51,7 @@ type probes struct {
 // Init initializes a Probes interface
 func Init(module *bpf.Module, netEnabled bool) (Probes, error) {
 
-	binaryPath, err := os.Executable()
-	if err != nil {
-		return &probes{}, fmt.Errorf("error getting tracee-ebpf path: %v", err)
-	}
+	binaryPath := "/proc/self/exe"
```

Comment:
@yanivagman as you said, we dont need CAP_DAC_OVERRIDE by using /proc/self/exe so we're good.